css-conditional-5/Overview.bs

```diff
@@ -1067,6 +1087,159 @@ Style Container Features</h3>
 	are [=computed value|computed=] with respect to the [=query container=],
 	the same as other values.
 
+<h3 id="scroll-state-container">
+Scroll State Container Features</h3>
+
+	A <dfn export>container scroll-state query</dfn> allows to query a container for various
+	state that depends on scroll position. The ''scroll-state'' [=query container=] can either
```

Comment:
> This feels like it's trying to be a definition of what the query container is in this case, but it doesn't feel like it gives a clear definition. I think it either (a) needs to be a clear definition or (b) needs to sound less like it's trying to define what the query container is. (I think (b) is the right thing because it's defined above, although I'm not sure.)

I changed the text to focus on scroll-state queries instead of the query container.